File: duplicate-assert/python-unittest/FileLengthCheckTest_testExtensions.py
import unittest

class FileLengthCheckTest(unittest.TestCase):

    def test_extensions(self):
        check = FileLengthCheck()
        check.set_file_extensions("java")
        self.assertEqual(".java", check.get_file_extensions()[0], "extension should be the same")
        check.set_file_extensions(".java")
        self.assertEqual(".java", check.get_file_extensions()[0], "extension should be the same")
        with self.assertRaises(IllegalArgumentException) as context:
            check.set_file_extensions(None)
        self.assertEqual("Extensions array can not be null", str(context.exception), "Invalid exception message")

File: duplicate-assert/java-junit/FileLengthCheckTest_testExtensions.java
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.fail;

import org.junit.jupiter.api.Test;

public class FileLengthCheckTest extends AbstractModuleTestSupport {

    @Test
    public void testExtensions() {
        final FileLengthCheck check = new FileLengthCheck();
        check.setFileExtensions("java");
        assertEquals(".java", check.getFileExtensions()[0], "extension should be the same");
        check.setFileExtensions(".java");
        assertEquals(".java", check.getFileExtensions()[0], "extension should be the same");
        try {
            check.setFileExtensions((String[]) null);
            fail("IllegalArgumentException is expected");
        }
        catch (IllegalArgumentException ex) {
            assertEquals("Extensions array can not be null", ex.getMessage(),
                    "Invalid exception message");
        }
    }
}
